pandas/tests/arrays/integer/test_concat.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+
+
+@pytest.mark.parametrize(
+    "to_concat_dtypes, result_dtype",
+    [
+        (["Int64", "Int64"], "Int64"),
```

Comment:
what about nullable int with numpy int -> object?

Reply:
I think it can probably return nullable int instead of object dtype, but see https://github.com/pandas-dev/pandas/pull/33607/files#r410127707, fleshing out the exact behaviour for concat with nullable integers is for a follow-up